src/RasterDataSources.jl

```diff
@@ -16,23 +16,50 @@ export WorldClim, CHELSA, EarthEnv, AWAP, ALWB
 
 export BioClim, Climate, Weather, LandCover, HabitatHeterogeneity
 
+# RCP
+export RCP26, RCP45, RCP60, RCP85
+export SSP126, SSP245, SSP370, SSP585
+
+# CC models from CMIP5 (used in CHELSA)
+export ACCESS1, BNUESM, CCSM4, CESM1BGC, CESM1CAM5, CMCCCMS, CMCCCM, CNRMCM5,
```

Comment:
I think dispatching on values is awkward - and it also requires the big `elseif` (like I have to do for years), so it's overall less maintainable. I think the exports are not too bad, since the names are really quite specific and it's unlikely that users would have variables like that. 

The alternative is... a sub-module named `FutureClimate` and we call `FutureClimate.CanEMS5`? It seems a bit more verbose than we need.

Reply:
No worries, there are for and against for both.